src/lib/utils/utils.js

```diff
@@ -4,6 +4,21 @@ const {canonicalHost} = require('./host');
 
 const balanceRegex = /([0-9]+) ?([a-zA-Z]*)/;
 
+const ZERO_ADDRESS_SHORTHAND_REGEX = /^0x0$/;
+const ZERO_ADDRESS_SHORTHAND_SEARCH_REGEX = /'0x0'/g;
+const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
+
+function extendZeroAddressShorthand(value) {
+  if (typeof value === 'string' && value.match(ZERO_ADDRESS_SHORTHAND_REGEX) !== null) {
+    return ZERO_ADDRESS;
+  }
+  return value;
+}
+
+function replaceZeroAddressShorthand(value) {
+  return value.replace(ZERO_ADDRESS_SHORTHAND_SEARCH_REGEX, `'${ZERO_ADDRESS}'`);
+}
+
```

Comment:
As we discussed yesterday utils becomes a monster slowly but surely.
Can it be in:
`src/lib/utils/addressUtils.ts`

Also note how I suggest typescript at the same time ;)